DigitalLearningSolutions.Data/Services/NotificationPreferencesDataService.cs

```diff
@@ -59,18 +65,23 @@ public IEnumerable<NotificationPreference> GetNotificationPreferencesForDelegate
                 return new List<NotificationPreference>();
             }
 
-            return connection.Query<NotificationPreference>(
-                @"DECLARE @Whitespace VARCHAR(3)
-                    SET @Whitespace = CHAR(10)+CHAR(13)+CHAR(32)
-
-                    SELECT DISTINCT n.NotificationID, TRIM(@Whitespace FROM n.NotificationName) AS NotificationName, n.Description, CASE WHEN nu.NotificationUserID IS NOT NULL THEN 1 ELSE 0 END AS accepted
+            var preferences = connection.Query<NotificationPreference>(
+                @"SELECT DISTINCT n.NotificationID, n.NotificationName, n.Description, CASE WHEN nu.NotificationUserID IS NOT NULL THEN 1 ELSE 0 END AS accepted
 	                    FROM Notifications AS n
 	                    JOIN NotificationRoles AS nr ON nr.NotificationID = n.NotificationID
 	                    LEFT JOIN NotificationUsers AS nu ON nu.NotificationID = n.NotificationID AND nu.CandidateID = @delegateId
 	                    WHERE nr.RoleID = 5
                         ORDER BY n.NotificationID",
                 new { delegateId }
-            );
+            ).ToList();
+
+            foreach (var notificationPreference in preferences)
+            {
+                // the notification names in the DB have trailing whitespace
+                notificationPreference.NotificationName = notificationPreference.NotificationName.Trim();
+            }

```

Comment:
Super minor, but could we extract this into a private RemoveTrailingSpace method to avoid duplication (especially because comments tend to get outdated pretty quickly)?